modules/nw-metallb-operator-setting-pod-priority-affinity.adoc

```diff
@@ -5,26 +5,28 @@
 [id="nw-metallb-operator-setting-pod-priority-affinity_{context}"]
 = Configuring pod priority and pod affinity in a MetalLB deployment
 
-You can optionally assign pod priority and pod affinity rules to `controller` and `speaker` pods by configuring the `MetalLB` custom resource. The pod priority indicates the relative importance of a pod on a node and schedules the pod based on this priority. Set a high priority on your `controller` or `speaker` pod to ensure scheduling priority over other pods on the node. 
+You can optionally assign pod priority and pod affinity rules to `controller` and `speaker` pods by configuring the `MetalLB` custom resource. The pod priority indicates the relative importance of a pod on a node and schedules the pod based on this priority. Set a high priority on your `controller` or `speaker` pod to ensure scheduling priority over other pods on the node.
 
-Pod affinity manages relationships among pods. Assign pod affinity to the `controller` or `speaker` pods to control on what node the scheduler places the pod in the context of pod relationships. For example, you can allow pods with logically related workloads on the same node, or ensure pods with conflicting workloads are on separate nodes. 
+Pod affinity manages relationships among pods. Assign pod affinity to the `controller` or `speaker` pods to control on what node the scheduler places the pod in the context of pod relationships. For example, using pod affinity rules, you can ensure that these pods are co-located on the same nodes or nodes. This can help optimize network communication and reduce latency between these components.
```

Comment:
```suggestion
Pod affinity manages relationships among pods. Assign pod affinity to the `controller` or `speaker` pods to control on what node the scheduler places the pod in the context of pod relationships. For example, by using pod affinity rules, you can ensure that these pods are co-located on the same nodes or different/separate nodes. This can help optimize network communication and reduce latency between these components.
```

Reply:
![Screenshot from 2023-08-31 12-02-44](https://github.com/openshift/openshift-docs/assets/57954076/93b15c8b-7f26-4d00-a6fb-b49cc447a871)

Reply:
Ah yes, now I see: same node versus same nodes